cmd/notation/sign.go

```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 	"time"
 
+	"context"
+
```

Comment:
please make the import tidy by removing the extra line and group the built-in import.

Reply:
Updated.